content/en/docs/ibm/create-cluster.md

```diff
@@ -44,12 +44,28 @@ Get the Kubeconfig file:
 ibmcloud ks cluster config --cluster $CLUSTER_NAME
 ```
 
-From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow).
+From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow-on-iks).
 
 
 ## Create and setup a new cluster
 
-Follow these steps to create and setup a new [IBM Cloud Kubernetes Service(IKS) cluster:
+Follow these steps to create and setup a new IBM Cloud Kubernetes Service(IKS) cluster:
+
+### Choose between a `classic` or `vpc-gen2` provider.
+A `vpc-gen2` cluster does not expose each node to the public internet directly and thus has more secure
+and more complex network setup. A `classic` provider, does expose each cluster node to the public internet, and thus has
+simpler networking setup - it might be a good option for trial run of Kubeflow.
```

Comment:
```suggestion
and more complex network setup. A `classic` provider does expose each cluster node to the public internet and therefore has
a relatively simpler networking setup - it might be a good option for you for a trial Kubeflow run.
```